jest.config.js

```diff
@@ -40,16 +40,10 @@ config.coverageThreshold = {
         lines: 56
     },
     './*/message-bus/**/*.js': {
-        statements: 15,
-        branches: 60,
-        functions: 0,
-        lines: 15
-    },
-    './*/request/**/*.js': {
-        statements: 23,
-        branches: 60,
-        functions: 0,
-        lines: 23
+        statements: 100,
+        branches: 100,
+        functions: 100,
+        lines: 100
```

Comment:
The coverage for the message bus and request service are now well above the default, they don't need explicit limits anymore. Consider removing lines 36-48 instead.